charts/apiclarity/templates/deployment.yaml

```diff
@@ -79,8 +89,13 @@ spec:
               value: kubernetes
             - name: FUZZER_IMAGE_NAME
               value: {{ .Values.apiclarity.modules.fuzzer.docker.image }}
+            {{- if .Values.apiclarity.tls.enabled }}
+            - name: FUZZER_PLATFORM_HOST_FROM_FUZZER
+              value: http://{{ include "apiclarity.name" . }}.{{ .Release.Namespace }}:8080/api
```

Comment:
if else same values, should it be https?

Reply:
it is on purpose for now, we need that the fuzzer will support it first and then we will change to to talk to the TLS endpoint

Reply:
worth adding a todo comment